app/components/course_component.rb

```diff
@@ -16,8 +17,6 @@ def initialize(course:, filter:)
   end
 
   def get_course_tag(programme)
-    return "#{programme} certificate" if %w[Primary Secondary].include?(programme)
-
-    programme
+    display_programme_tag programme
```

Comment:
Could you add :display_programme_tag to the "delegate" call to helpers, rather than including all of the ProgrammesHelper helpers